tsdb/points.go

```diff
@@ -987,7 +996,8 @@ type Fields map[string]interface{}
 
 func parseNumber(val []byte) (interface{}, error) {
 	for i := 0; i < len(val); i++ {
-		if val[i] == '.' {
+		// If there is a decimal or an N (NaN), parse as float
+		if val[i] == '.' || val[i] == 'N' || val[i] == 'n' || val[i] == 'I' || val[i] == 'i' {
```

Comment:
`Inf`.  It's a valid value for `strconv.ParseFloat`.  It was causing a panic previous because it would fall-through to the boolean parsing.